tests/test_datasets/test_dataset.py

```diff
@@ -874,3 +903,70 @@ def make_samples(df):
     np.testing.assert_array_equal(
         torch_dataset[1]["target"], tsdf_with_exog.df.loc[:, pd.IndexSlice["Omsk", "target"]].values
     )
+
+
+@pytest.mark.parametrize(
+    "expected_columns",
+    ((["target", "regressor_1", "regressor_2"]),),
+)
+def test_update_columns_update_existing_columns(df_and_regressors, expected_columns):
```

Comment:
We don't check that values are changing, I think we should add test on it. For example, fill columns with zero.

Reply:
Done